.claude/settings.json

```diff
@@ -0,0 +1,15 @@
+{
+  "hooks": {
+    "PostToolUse": [
+      {
+        "matcher": "Write|Edit|MultiEdit",
+        "hooks": [
+          {
+            "type": "command",
+            "command": "FILE_PATH=$(jq -r '.tool_input.file_path'); if echo \"$FILE_PATH\" | grep -q '\\.swift$'; then echo \"Running SwiftLint on $FILE_PATH\"; swift package --package-path BuildTools plugin --allow-writing-to-package-directory --allow-writing-to-directory . swiftlint --config ../.swiftlint.yml --fix \"$FILE_PATH\" && echo \"SwiftLint completed for $FILE_PATH\" || echo \"SwiftLint failed for $FILE_PATH\"; fi"
```

Comment:
> swift package --package-path BuildTools plugin --allow-writing-to-package-directory --allow-writing-to-directory .

This loads up the swiftlint version from our `BuildTools` so you shouldn't need to install anything else. The first time it runs things may be a bit slow but after install it should work.